internal/resource/configvalidators/whitespace_validator.go

```diff
@@ -0,0 +1,43 @@
+package configvalidators
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+var _ validator.String = &whitespaceValidator{}
+
+type whitespaceValidator struct{}
+
+func (v whitespaceValidator) Description(ctx context.Context) string {
```

Comment:
I would rename this to imply that whitespace isn't allowed

Reply:
renamed to noWhitespaceValidator and changed file name